### ISSUE TITLE
Small Toast Size

### ISSUE THREAD

**Issue by dsblack115** [nmsmith22389/HotLoot#23]
Setting to small toast size does not work.  Error attached.
![capture](https://user-images.githubusercontent.com/22847171/28753838-022c5b60-7508-11e7-91fa-4cd713725400.PNG)

**Comment by eviilwave** [comment]
```1x HotLoot\core.lua:1026: attempt to index field 'value' (a nil value)
HotLoot\core.lua:1026: in function `SetLoot'
HotLoot\core.lua:277: in function `TestLootMonitor'
HotLoot\Modules\options\options-Options.lua:632: in function <HotLoot\Modules\options\options.lua:631>
(tail call): ?
[C]: ?
[string "safecall Dispatcher[3]"]:9: in function <[string "safecall Dispatcher[3]"]:5>
(tail call): ?
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-61.lua:799: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:614>
(tail call): ?
[C]: ?
[string "safecall Dispatcher[4]"]:9: in function <[string "safecall Dispatcher[4]"]:5>
(tail call): ?
Masque\Libs\AceGUI-3.0\AceGUI-3.0-34.lua:314: in function `Fire'
...sque\Libs\AceGUI-3.0-34\widgets\AceGUIWidget-Button.lua:22: in function <...sque\Libs\AceGUI-3.0\widgets\AceGUIWidget-Button.lua:19>

Locals:
nil

**Comment by nmsmith22389** [comment]
Thanks for posting this! It's def something I will fix! 😄

**Comment by kp95535** [comment]
I had the same issue and found that the Text tab had the Value Enabled and that was what was causing the issue. (The description text for the Enable check box has the "Large size theme only" displayed but was checked by default) I never thought to look at that just happened upon it.

**Comment by dsblack115** [comment]
I had a couple options that were large toast size only, disabled them, and now the small toast size is working.

**Comment by nmsmith22389** [comment]
Fixed in **v3.3** which is coming in the next day!!! 🎊 😀 There was a typo in the code that only showed up in the small size.
